internal/ld/ld.go

```diff
@@ -42,10 +40,10 @@ type ApiOptions struct {
 }
 
 const (
-	apiVersion       = "20210729"
+	apiVersion       = "20220603"
```

Comment:
@jazanne this looks good to me, but I don't know the status of the API version now. Are we handling the pagination easily at this point?

Reply:
pagination still isn't required (forthcoming in "v4")